language/move-prover/bytecode-to-boogie/tests/driver/mod.rs

```diff
@@ -50,7 +50,8 @@ pub fn generate_boogie(file_name: &str, target_modules: &[&str]) -> String {
     file_names.push(file_name);
     let (modules, source_maps) = compile_files(file_names.to_vec());
 
-    if cfg!(feature = "golden") {
+    let golden_option = option_env!("VERIFY_BPL_GOLDEN");
+    if golden_option.is_some() && golden_option.unwrap() == "1" {
```

Comment:
Nit: pattern matching or some option API to avoid unwrap? E.g.,

if option_env!("VERIFY_BPL_GOLDEN").unwrap_or("0") == "1" {

Reply:
Done.